README.md

```diff
@@ -367,6 +368,11 @@ The following descriptions provide additional elaboration on a few common parame
 | `--exported` | Use `mockery --exported` to generate public mocks for private interfaces. |
 | `--with-expecter` | Use `mockery --with-expecter` to generate `EXPECT()` methods for your mocks. This is the preferred way to setup your mocks. |
 
+Excluding Directories
+---------------------
+
+Directories can be excluded from mock generation by placing a file `.mockery_skip` inside them. Mocks will not be generated for any types present in a directory containing such a file nor for any in its sub-directories.
```

Comment:
Actually, let's call this `.mockery_skip_all`. I envision we might want a `.mockery_skip` that skips _just_ a single directory, whereas the `_all` can skip the entire tree.

Reply:
By "skip the entire tree" vs "just a single directory" is the intended meaning that
- "skip the entire tree" is recursively down the tree from this point
- "just a single directory" is don't generate mocks for files in this directory but do for those in directories beneath this one
? (checking I've got that right in my head)

Reply:
Yep that sounds right, I feel like we might want both options at some point but we can just focus on "skip the entire tree" as you were originally doing. Thanks!